src/platforms/spark/SparkMediaplayer.mjs

```diff
@@ -1,10 +1,21 @@
 import lng from "wpe-lightning/src/lightning.mjs";
 
-export default class SparkMediaplayer extends lng.Component {
+var appVideoEl = null;
+
+// clear video element resources
+export function clearVideoElementResources() {
```

Comment:
this duplicates **sparkscene.on('onClose'** handler in SparkMediaplayer.

Reply:
yes, updated

Reply:
I think a better approach is to use **sparkscene.on('onClose'** and avoid static global variables